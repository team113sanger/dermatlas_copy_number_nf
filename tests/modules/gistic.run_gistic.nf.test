nextflow_process {

    name "Test Process RUN_GISTIC2"
    script "modules/gistic2.nf"
    process "RUN_GISTIC2"
    tag "local"

    test("Should run without failures") {

        when {
            params {
                load("$baseDir/tests/testdata/test_params.json")
            }
            process {
                """
                input[0] = tuple([analysis_type: "test", plot_dir: "test"], 
                                file("${baseDir}/tests/testdata/one_tumor_per_patient.tsv"))
                input[1] = file("${params.gistic_refgene_file}")
                """
            }
        }

        then {
            assert process.success
        }

    }

}
