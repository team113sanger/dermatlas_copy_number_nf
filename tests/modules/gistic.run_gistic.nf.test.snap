{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "all_lesions.conf_95.txt:md5,f8e6f213fdee100d31b07138c9e95a4d"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-05-21T11:18:26.741312"
    }
}